src/types.rs

```diff
@@ -831,6 +831,11 @@ impl Context {
     /// };
     ///
     /// assert_eq!(example_context.display(&sig), "A([!] x_)".to_string());
```

Comment:
I don't see what this fourth test adds that the fifth test wouldn't show better if we added a variable to that term. If you agree, could you remove the fourth test and update the fifth test accordingly? Actually removing tests 1-4 and just using this fifth test for a term like `x_ [!] A [2, 1, 0]` might work well. If you agree, please update.